samples/star-wars-api/Program.fs

```diff
@@ -2,13 +2,39 @@ namespace FSharp.Data.GraphQL.Samples.StarWarsApi
 
 open Microsoft.AspNetCore
 open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.Configuration
+
 
 module Program =
     let exitCode = 0
 
-    let buildWebHost args =
+    let buildWebHost (args: string array) =
+
+        // Build an initial configuration that takes in both environment and command line settings.
+        let config = ConfigurationBuilder()
+                            .AddEnvironmentVariables()
+                            .AddCommandLine(args)
+                            .Build()
```

Comment:
As we use default builder
https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webhost.createdefaultbuilder?view=aspnetcore-6.0